neonvm-runner/cmd/main.go

```diff
@@ -487,17 +488,42 @@ func runQEMU(
 
 	callbacks = cpuServerCallbacks{
 		get: func(logger *zap.Logger) (*vmv1.MilliCPU, error) {
-			return lo.ToPtr(vmv1.MilliCPU(lastValue.Load())), nil
+			switch cfg.cpuScalingMode {
```

Comment:
I think nesting here is a bit too much, IMO we should change the structure of code here.

Reply:
Agreed in DM to proceed with that later